docs/en/02_Developer_Guides/19_GraphQL/01_getting_started/05_deploying_the_schema.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Deploying the schema
+summary: Deploy your GraphQL schema to a test or production environment
+icon: rocket
+---
+
+# Getting started
+
+[CHILDREN asList]
+
+[alert]
+You are viewing docs for a pre-release version of silverstripe/graphql (4.x).
+Help us improve it by joining #graphql on the [Community Slack](https://www.silverstripe.org/blog/community-slack-channel/),
+and report any issues at [github.com/silverstripe/silverstripe-graphql](https://github.com/silverstripe/silverstripe-graphql).
+Docs for the current stable version (3.x) can be found
+[here](https://github.com/silverstripe/silverstripe-graphql/tree/3)
+[/alert]
+
+## Deploying the schema
+
+One way or another, you must get the `.graphql-generated/` and `public/_graphql/` folders into your test and production environments for Silverstripe CMS (and your own custom queries) to work as expected. There are many ways to do so. The options below are listed in order of complexity.
+
+### Options for any hosting solution
+
+#### Commit the schema to version control {#commit-to-vcs}
```

Comment:
Putting this option first make it seem like it's the recommendation.  It's not, it's more like an option of last resort.  Better to start with 'on-demand' IMO since it's suitable for the majority of sites, and then offer other solution later for those that require the optimisation

Reply:
> The options below are listed in order of complexity

This has been very clearly stated.

Reply:
Do nothing and rely on the fallback is the least complex option so it should be first then?

Reply:
That's in the "Multi-server hosting solutions" section, which makes it inherently more complex than the "Options for any hosting solution" approaches.

Reply:
Talked offline - tl;dl

Add a section up the top saying:

"If your setup is a single webserver and run dev/build as part of your deployment pipeline, then you don't need to do anything. If you setup is something else, you'll need to implement one of these options."

Reply:
Done. Also fixed a typo (graphsl should be graphql)